agent/src/cli.rs

```diff
@@ -36,10 +36,21 @@ pub struct Arguments {
     #[clap(short = 's', long = "server-url", default_value_t = DEFAULT_SERVER_ADDRESS.parse().unwrap())]
     /// The server url.
     pub server_url: Url,
-
     /// An existing path where to manage the fifo files.
     #[clap(short = 'r', long = "run-folder", default_value_t = DEFAULT_RUN_FOLDER.into())]
     pub run_folder: String,
+    #[clap(long = "insecure", default_value_t = false)]
+    /// Flag to disable TLS communication between Ankaios agent and server.
+    pub insecure: bool,
+    #[clap(long = "ankaios_agent_ca_pem", env)]
```

Comment:
I would keep the long names rather shorter. "ankaios_agent" as name is not required here as all arguments belong to the agent. So what about --ca_pem as argument with correponding `ANKAGENT_CA_PEM` as environment variable?